README.md

```diff
@@ -1,56 +1,37 @@
 # `ayb`
-With `ayb`, all your base can finally belong to you. Move SQL for great justice.
+Ayb is the easiest way to get started writing shareable reproducable database demo apps.  With only 5 commands you can have a sharable database.
 
 [![Build status](https://github.com/marcua/ayb/actions/workflows/tests.yml/badge.svg)](https://github.com/marcua/ayb/actions/workflows/tests.yml)
 
 
 ## Introduction
 
```

Comment:
Generally in READMEs I like to first explain/demonstrate the benefit of a piece of software, then jump into installation instructions.  Reason being is that I don't care how to install a piece of software until I understand what it does for me.

You are doing a bit of that... but one of the biggest selling points of ayb is the ease of installation of an understood concept - a relational database.  So maybe in this case you'd want to move up the install commands even earlier, definitely shorten them.